### DISCUSSION TITLE
Remix is spooked by its own changes

### DISCUSSION THREAD

**Discussion by mushketyk** [remix-run/remix#3826] [Q&A] [answered]
The desktop version of Remix has an odd behavior when editing files in a local git repository.

If I edit a smart contract code, after a few edits it displays the following modal page complaining about the file that I've edited:

![image](https://user-images.githubusercontent.com/592286/180624872-6ce84938-29e6-49f6-b567-626837c99647.png)

This behavior is quite consistent and I get this error every time after I make a few changes. If I click on any of the buttons, the modal reappears again after a few more changes.

There are no other processes editing this file in the background. So the only explanation that I can think of is that Remix does not recognize its own changes and displays this warning for its own edits.

**Software versions:**

Remix version: Version 1.3.4 (1.3.4.825)
OS: macOS 12.0.1 (21A559)

How to reproduce:

1. Open Remix locally.
2. Click "Connect to Localhost"
3. Open a folder with a git repository
4. Select workspace "localhost"
5. Try to make a few edits in a smart contract

**Comment by mushketyk** [comment]
Moved to an issue: https://github.com/remix-run/remix/issues/3827

**Comment by MichaelDeBoey** [comment, marked as answer]
Hello 👋
I think you may be in the wrong place. This GitHub repo is all about the Remix Web Framework which you can learn about at https://remix.run.
Closing this as you have mixed this up with the Remix IDE (https://remix-project.org) which is a completely different project.
